FrameRate/FrameRate.cpp

```diff
@@ -18,14 +18,20 @@
 **/
 
 #include "FrameRate.h"
-
+#include "host.hpp"
+#include "exception.hpp"
 #include "utils.h"
+#include "dsError.h"
 
 // Methods
 #define METHOD_SET_COLLECTION_FREQUENCY "setCollectionFrequency"
 #define METHOD_START_FPS_COLLECTION "startFpsCollection"
 #define METHOD_STOP_FPS_COLLECTION "stopFpsCollection"
 #define METHOD_UPDATE_FPS_COLLECTION "updateFps"
+#define METHOD_SET_FRAME_MODE "setfrmmode"
```

Comment:
Pls change these to camelCase.
setFrmMode
getFrmMode
setDisplayFrameRate
getDisplayFrameRate